backend/fms_core/tests/valid_templates/Project_link_samples_v3_4_0_B_A_1.csv

```diff
@@ -0,0 +1,1001 @@
+﻿Project link samples Template,,,,,
+,,,,,
+Instructions:,,,,,
+,Each line define the addition or removal of a sample instance from a Project. Each sample instance will ,,,,
+,inherit from its parent’s association when extracted or transfered. A sample instance can be associated,,,,
+,to more than one project.,,,,
+,,,,,
+#,Action,Project Name,Sample Name,Sample Container Barcode,Sample Container Coord
+1,Add,TestProject1,sample1,tube001,
+2,Add,TestProject2,sample2,tube002,
+3,,,,,
+4,,,,,

```

Comment:
1000 lines of csv ... is this a stress test? :)

Reply:
Right, I assume you probably generated your csv automatically from an xlsx file; so you can remove the extra lines at the end of your csv